doc/api/videoThumbnailLoader.md

```diff
@@ -0,0 +1,132 @@
+# VideoThumbnailLoader #########################################################
+
+
+## Overview ####################################################################
+
+The VideoThumbnailLoader is a tool that can help using a video track as a
+substitute of a manifest embedded thumbnail track.
+
+The goal is to make a thumbnail out of HTML5 video element, by :
+- Managing the loading / appending of resources from a given track
+(video segments).
+- Exploiting the Media Source Extension API to make it invisible to user.
+
+The tool will need the loaded manifest to contain trickmode tracks. These kind
+of adaptation exists in MPEG-DASH and HLS, and contains lightweight video tracks,
+most of the time including one unique frame for each video segments. As video
+segments from trickmode tracks may be quicker to load and easier to decode, they
+are preferred over standard video tracks for creating thumbnails.
+
+## How to use it ###############################################################
+
+
+As an experimental tool, the VideoThumbnailLoader won't be included in a
+default RxPlayer build.
+
+Instead, it should be imported by adding the RxPlayer through a dependency
+trough the npm registry (e.g. by doing something like ``npm install
+rx-player``) and then specifically importing this tool from
+``"rx-player/experimental/tools"``:
+
+```js
+import { VideoThumbnailLoader } from "rx-player/experimental/tools";
+import RxPlayer from "rx-player";
+
+const player = new RxPlayer({ /* some options */ });
+player.loadVideo({ /* some other options */ });
+
+// Video element used to display thumbnails.
+const thumbnailVideoElement = document.createElement("video");
+const videoThumbnailLoader = new VideoThumbnailLoader(
+  thumbnailVideoElement,
+  player
+);
+
+// Ask for the video thumbnail loader to fetch the thumbnail
+// that should be displayed at presentation time = 200.
+videoThumbnailLoader.setTime(200);
+```
+
+## Functions ###################################################################
+
+### addFetcher #################################################################
```

Comment:
I don't find the word `fetcher` very comprehensible there.

Why not just "loader" (already in the name of the VTL)